hugo/content/installation-plus.md

```diff
@@ -0,0 +1,453 @@
+---
+title: "Install NGINX Agent From NGINX Plus Repository"
```

Comment:
Title to change to "Installation from NGINX Plus Repository" to be in line with the OSS title.